### DISCUSSION TITLE
Can't log in with GitHub account

### DISCUSSION THREAD

**Discussion by M-elhasan9** [Laravel-Backpack/community-forum#954] [Q&A (Help)]
I had an account with this email before, and I have another account where I bought a Backpack Pro . I linked the paid account to the personal email associated with the GitHub account. Now, when I try to log in through GitHub, I can't, and an error message appears.

**Comment by karandatwani92** [comment]
Hey @M-elhasan9 

@tabacitu can help you here. Can you share the problem at `hello@backpackforlaravel.com`?

Sorry for the inconvenience!

**Comment by tabacitu** [comment]
I did see like 10 failed login attempts yesterday with Github, so I'm going to assume it was you 😀 Yes, like Karan said, please email me. 

If you logged in with Github and lost access tot that account... you should be able to [reset your password](https://backpackforlaravel.com/password/reset). That way you can start logging in with email & password.
